editor.planx.uk/src/ui/shared/InputRowLabel.tsx

```diff
@@ -5,6 +5,7 @@ import React, { ReactNode } from "react";
 const Label = styled(Typography)(({ theme }) => ({
   flexShrink: 1,
   flexGrow: 0,
+  width: "100%",
```

Comment:
![image](https://github.com/theopensystemslab/planx-new/assets/20502206/38173b8a-7deb-43c6-b8ca-dcd966f343cc)

It looks like this change has some unintended consequences in Design Settings where this component is also used.

@ianjon3s is going to take a wider review of these components and labels, so for now we could (temporarily) keep them inline, or use a repeated `<Typography component="label" variant="subtitle2">`

Reply:
Another example I just hit - 
![image](https://github.com/theopensystemslab/planx-new/assets/20502206/93f545db-1ed7-43d5-8b83-61e969f55c3b)